.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+dist: trusty
```

Comment:
Please use xenial (16.04) instead of trusty.

Reply:
@fsaad there is currently no xenial build environment on travis: https://docs.travis-ci.com/user/reference/overview/

will get back to you on the updated versions for meta.yaml shortly